vllm/distributed/device_communicators/pynccl_utils.py

```diff
@@ -40,7 +40,8 @@ def set_pynccl_stream(stream: torch.cuda.Stream):
 def init_process_group(group: Optional[ProcessGroup] = None) -> None:
     assert not is_initialized()
     global comm
-    logger.info("vLLM is using nccl==%s", ncclGetVersion())
+    logger_data = {"nccl": ncclGetVersion()}
+    logger.info("vLLM is using nccl==%s", ncclGetVersion(), extra=logger_data)
```

Comment:
I'd suggest using a variable for the result of `ncclGetVersion` and using the variable for the dict and the log message.

```suggestion
    nccl_version = ncclGetVersion()
    logger_data = {"event": "nccl-version", "version": nccl_version}
    logger.info("vLLM is using nccl==%s", nccl_version, extra=logger_data)
```